Week1-Coding-Task.hs

```diff
@@ -0,0 +1,76 @@
+--Section 1, credit card validation
+--Double value of every second digit from beginning of right 
+--Add digits of doubled and undoubled values to form new number 
+--Calculate result mod 10: if 0, valid 
+
+------------------------------------------------------------------------------
+--Exercise 1: toDigits and toDigitsRev 
+------------------------------------------------------------------------------
+toDigits :: Integer -> [Integer]
+toDigitsRev :: Integer -> [Integer]
+
+--A list with elements reversed, so can use it later for reversing toDigits 
+rev :: [Integer] -> [Integer]
+rev [] = []--base case 
+rev (s:l) = rev l ++ [s]--step case
+
+toDigits s 
+    | s <= 0 = []
+    | s < 10 = [s] -- They have only one digit, so one element list 
+    | otherwise = toDigits (s`div`10) ++ [s`mod`10]--mod10 results only give one digit  
```

Comment:
According to our style guide (and it's also a common practice in Haskell) you should add spaces around `div` and `mod` like this:

```haskell
toDigits (s `div` 10) ++ [s `mod` 10]
```